posts/start-web-performance-first-steps.md

```diff
@@ -0,0 +1,120 @@
+- [In Summary](#in-summary)
+- [See Also](#see-also)
+
+Also, this post is focused on paying performance technical debt, but all these steps can be implemented from day one and actually help us shape a better product from the get-go.
+
+## The Challenges
+
+Already familiar with the challenges of tackling performance technical debt? [Go straight to the steps ⏭](#the-steps)
+
+
+## The Steps
+
+Given the set of challenges we faced and our experience, we've assembled 3 simple steps that we can do start working on performance:
+
+- Measure and Lock: Use real user metrics and lab metrics to prevent degradation, to understand how performance affects your business and to pass knowledge forward;
+- Set up a group: Gather individuals that are interested in the subject and give them support and incentive to keep the discussion alive, evolve and maintain the tools;
+- Make performance a part of the Company's Culture: Build tools to lower the barriers and incetivize the Engineering team, share learnings and results to engage the product, design and business areas of the company
+
+There are a lot of other things we did and each of this point merits a post of it's own, but we'll focus here on explaining the goal and general idea of each. Let me know if you'd like to a more in depth explanation of the tools we used and built to address each of these steps!
+
+## Step One: Measure and Lock
+
+The main goals are three-fold:
+
+1. Understanding how performance impacts our business: [Measure](#measure-real-user-metrics-and-conversion-metrics)
+2. Preventing silent degradation beyond a point: [Measure](#measure) and [Lock](#lock-audits-and-continous-improvement)
+3. Automatizing the knowledge transfer: [Lock](#lock-audits-and-continous-improvement)
+
+### Measure: Real User Metrics and Conversion Metrics
+
+We've always had an underlying feeling that performance was an issue for our users and that we would improve their experience if we optimized our product but, before measuring, we failed to understand just how much that is.
+
+Since we started measuring and tagging our users based on the performance, we've been able to better understand how their behavior are impacted by the performance they had when seeing and using a feature.
+
+[We've written about this](/posts/avoid-losing-users-and-conversions.md) and also [there's a lot of relevant content about it](#see-also), so I'll just leave the main points here:
+
+1. Start collecting [Real User Metrics](https://web.dev/user-centric-performance-metrics/) as soon as possible;
+2. Track this in a way you can correlate it with your business and user metrics;
+3. Don't use only average and median a lot of your users are better described by the higher percentiles (p75, p90, p99, p99.99);
+4. Build discoverable, accessible and easy to use dashboards showing this metrics and how they relate to your business.
```

Comment:
I missed:
* a topic about lab metrics and why it's useful
* the "how" we measure them (lab and RUM). An article about tooling may be better to deep dive, but you could have a small paragraph here introducing this article (to come) with a lit bit of spoilers, citing SpeedCurve, amplitude and buckets strategy and other js performance APIs

Reply:
I added it to the `Lock` part :) On the "How", I think we should write that on a future article, WDYT?
 
I am not sure what's the best idea here if we should actually publish the ones related to each of the detailed steps first rather than later 🤔